ci/docker/Dockerfile.maldet.centos.7

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018-2022 Intel Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build an environment scan DAOS for malware.
+#
+
+# Pull base image
+FROM centos:centos7
```

Comment:
Please do not add any more CentOS 7 code.  We can safely remove building CentOS 7 rpms on master entirely.

Reply:
We can not remove it yet.   We still have active release branches that support CentOS 7.9 and reports of DAOS on CentOS 7.9 active use in the field, and this change will need to be cherrypicked into them.

Reply:
We are removing all Centos 7 support from master, and that includes building and testing RPMs etc.  Adding this change to other branches where we still support CentOS 7 will have to be a backport not a cherry-pick and you can add in the CentOS 7 support at that stage, but please remove all the CentOS 7 specific code from this PR.

Reply:
Removed in current push.